windows-nsis/openvpn.nsi

```diff
@@ -428,6 +428,11 @@ Function .onInit
 	${GetParameters} $R0
 	ClearErrors
 
+${IfNot} ${AtLeastWinVista}
+	MessageBox MB_OK "Windows Vista and above required"
+	Quit
+${EndIf}
+
```

Comment:
ok, I'll change to tabs

we might want to add some CI (??), which will check for tabs/spaces/etc